autoupdate.yaml

```diff
@@ -143,6 +148,18 @@
     Owner: kubernetes-csi
     Repository: livenessprobe
   Name: storage-livenessprobe
+- Name: traefik
+  Registry:
+    Images:
+    - SourceImage: library/traefik
+    Latest: true
+    VersionFilter: ^2.[0-9]+.[0-9]+$
+- Name: traefik3
```

Comment:
Shouldn't need to split these out. Just join the regexes together.

Reply:
Makes sense 👍

Reply:
I'm not sure this is a good idea. Merging the two entries, combined with using the `Latest` option, means that only the latest i.e. 3.x versions will get autoupdated. Traefik 2.11 is [supported until Feb. 2026](https://doc.traefik.io/traefik/deprecation/releases/). We want to mirror those 2.11 patch versions, right?